docker-experimental-swarm-cluster/azuredeploy.json

```diff
@@ -0,0 +1,254 @@
+            }
+        },
+        "virtualNetworkPrefix": {
+            "type": "string",
+            "defaultValue": "10.0.0.0/8",
+            "metadata": {
+                "description": "The IP range on CIDR notation of the virtual network where the resource will be deployed"
+            }
+        },
+        "subnetSwarmName": {
+            "type": "string",
+            "defaultValue": "Swarm-Subnet",
+            "metadata": {
+                "description": "The name of the virtual network where the swarm nodes will be deployed"
+            }
+        },
+        "subnetSwarmPrefix": {
+            "type": "string",
+            "defaultValue": "10.0.1.0/24",
+            "metadata": {
+                "description": "The IP range on CIDR notation of the virtual network where the resource will be deployed"
+            }
+        },
+        "subnetManageName": {
+            "type": "string",
+            "defaultValue": "Manage-Subnet",
+            "metadata": {
+                "description": "The name of the virtual network where the swarm nodes will be deployed"
+            }
+        },
+        "subnetManagePrefix": {
+            "type": "string",
+            "defaultValue": "10.0.0.0/24",
+            "metadata": {
+                "description": "The IP range on CIDR notation of the virtual network where the resource will be deployed"
+            }
+        },
+        "swarmNodesNumber": {
+            "type": "int",
+            "defaultValue": 3,
+            "metadata": {
+                "description": "Number of swarm cluster nodes to deploy"
+            }
+        },
+        "swarmClusterNodesSize": {
+            "type": "string",
+            "defaultValue": "Standard_A1",
+            "allowedValues": [
+                "Standard_A0",
+                "Standard_A1",
+                "Standard_A2",
+                "Standard_A3",
+                "Standard_A4"
+            ],
+            "metadata": {
+                "description": "Size of the VMs for the cluster nodes"
+            }
+        },
+        "adminUsername": {
+            "type": "string",
+            "defaultValue": "swarmAdmin",
+            "metadata": {
+                "description": "User name for the root user of the virtual machines"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "defaultValue": "Passw0rd!",
+            "metadata": {
+                "description": "Password of the root user account inside the virtual machines"
+            }
+        },
+        "swarmMasterPublicIpDnsName": {
+            "type": "string",
+            "defaultValue": "dockerswarmcluster",
+            "metadata": {
+                "description": "Name of the DNS assigned to the master node to be able to access through internet"
+            }
+
+        },
+        "swarmMasterPublicIpName": {
```

Comment:
This could become a variable to reduce the number of parameters a user has to enter